src/video/types.rs

```diff
@@ -26,6 +26,7 @@ pub struct CaptureConfiguration {
     pub height: u32,
     pub width: u32,
     pub frame_interval: FrameInterval,
+    pub thermal: bool,
```

Comment:
I think that we should expose such variable in a different way, maybe via a vector of enum of capabilities, or a struct.
The reason behind that is that we could add more fields, making the structure a bit confuse, also this is not a capture configuration option (as in the gstreamer pipeline), but as a camera capability thing. 
Maybe storing this in a total different structure may makes more sense, or a structure that encapsulates both.